pkg/meta/redis.go

```diff
@@ -1149,6 +1149,18 @@ func (r *redisMeta) Rmr(ctx Context, parent Ino, name string) syscall.Errno {
 	return r.emptyEntry(ctx, parent, name, inode, concurrent)
 }
 
+func (r *redisMeta) Info(ctx Context, parent Ino, summary* Summary) syscall.Errno {
+	if st := r.Access(ctx, parent, 3, nil); st != 0 {
+		return st
+	}
+
+	if st := r.Summary(ctx, parent, summary); st != 0 {
```

Comment:
The Summary() will walk through all the directories, and will not return the details of chunk/slices of a file.

I think we don't need to a new API, getattr and read should be enough

Reply:
ok, I will try.  
For the example you give, I don't know the meaning of the info. 
For the file `syslog`
`Inode ` is the inode number of the file or directory.
`directory ` is the directory numbers of the file
`files` is the chunk files? so the directory doesn't have the info?
`chunks` is the same meaning as the `files `?
`length` is the file or dir length which is the same as the results `os.stat` get
`size` is the file size or dir size, which is the same as the results `os.stat` get


```
/jfs/syslog:
 inodes: 	280
  directories: 	2
  files: 	278
 chunks: 	278
 length: 	654645174
 size: 		655237120
```

Reply:
It's not easy to know the total number of files and sub-dir, we can show the chunks/slices of a file.